generators/server/templates/quarkus/src/test/java/package/TestResources.java.ejs

```diff
@@ -23,10 +23,13 @@ import io.quarkus.test.common.QuarkusTestResource;
 import io.quarkus.test.h2.H2DatabaseTestResource;
 <%_ } _%>
 
+<%_ if (databaseType !== 'no') { _%>
```

Comment:
We should keep sql to avoid problem in the futur with noSql db

Reply:
I've updated the condition for `databaseType === 'sql'